modules/updating-sno.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assemblies:
+//
+// * updating/updating-cluster-between-minor.adoc
+// * updating/updating-cluster-cli.adoc
+// * updating/updating-cluster.adoc
+
+= Updating single node {product-title}
+
+You can update, or upgrade, a single-node {product-title} cluster using either the console or CLI. 
+
+However, note the following: 
+
+* The prerequisite to pause the `MachineHealthCheck` resources is not required, because there is no other node to perform the health check.
+
+* Restoring a single-node {product-title} cluster using an etcd backup is not officially supported. However, it is good practice to perform the etcd backup in case your upgrade fails. You might be able to restore your cluster to a previous state using the backup if your control plane is healthy.  
+
+* Updating a single-node {product-title} cluster requires downtime and can include an automatic reboot. The amount of downtime depends on the update payload: 
+
+** If the update payload contains an operating system update, which requires a reboot, the downtime is significant and impacts cluster management and user workloads. 
+
+** If the update contains machine configuration changes that do not require a reboot, the downtime is less and the impact on the cluster management and user workloads is lessened. In this case, the node drain is skipped with single-node {product-title} because there is no other node in the cluster to reschedule the workloads.
+ 
+** If the update payload does not contain an operating system update or machine configuration changes, a short API outage occurs and resolves quickly. 
+
+[IMPORTANT]
+====
+There are conditions, such as bugs in an updated package, that could cause the single node to not restart after a reboot. In this case, the update does not rollback automatically. 
```

Comment:
```suggestion
There are conditions, such as bugs in an updated package, that can cause the single node to not restart after a reboot. In this case, the update does not rollback automatically. 
```
Is there anything else helpful we can say here as super-basic troubleshooting or next steps?